src/main/kotlin/taxi/Elasticsearch.kt

```diff
@@ -0,0 +1,61 @@
+package taxi
+
+import com.natpryce.konfig.PropertyGroup
+import com.natpryce.konfig.getValue
+import com.natpryce.konfig.intType
+import com.natpryce.konfig.stringType
+import io.searchbox.client.JestClient
+import io.searchbox.client.JestClientFactory
+import io.searchbox.client.config.HttpClientConfig
+import io.searchbox.indices.CreateIndex
+import io.searchbox.indices.mapping.PutMapping
+import mu.KotlinLogging
+import org.apache.http.auth.AuthScope
+import org.apache.http.auth.UsernamePasswordCredentials
+import org.apache.http.impl.client.BasicCredentialsProvider
+
+
+private val logger = KotlinLogging.logger {}
+
+val passengerMapping =
```

Comment:
this looks messy, any way we can tidy this up or something? we'll probably have a few more mappings to come and bunches of these will make it a mess

Reply:
plus what about name and date of birth, we want to link this to google oauth right? name will be useful for the drivers and date of birth to verify age